website/src/styles/tokens.css

```
@@ -0,0 +1,35 @@
+  --text-secondary: #8892b0;
+  --text-muted: #5a6a8a;
+  --primary: #64ffda;
+  --primary-dim: #3d9e85;
+  --primary-bg: rgba(100, 255, 218, 0.06);
+  --primary-border: rgba(100, 255, 218, 0.18);
+  --success: #6bff8e;
+  --warning: #ffd93d;
+  --error: #ff6b6b;
+  --planned: #818cf8;
+  --code-bg: #0d1117;
+  --font-sans: "Inter", -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)